openedx/features/idea/models.py

```diff
@@ -78,11 +97,16 @@ def __unicode__(self):
         return self.title
 
     def toggle_favorite(self, user):
+        """
+        Toggle favorite status of idea.
+        :param user: User whose idea favorite status needs to be toggled
+        :return: boolean value regarding favorite status
+        """
```

Comment:
Just a reminder, that edx prefers Google style docstrings:

From edx docs:

Here’s how you write documentation in a mostly “Google Style” manner:

def func(arg1, arg2):
    """
    Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value
    """

Since it's not compulsory, we can keep it as it is. Just mentioning for future PRs.